src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -273,7 +273,7 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
       .thenCompose(funds -> {
         String fundId = HelperUtils.extractIdOfFirst(funds, "funds");
         if (StringUtils.isEmpty(fundId)) {
-          return lookupFundId(DEFAULT_LOOKUP_CODE);
+          return completedFuture(null);
```

Comment:
I believe the logic still can try to lookup by default fund code. The logic in this case should be something like 
```java
return DEFAULT_LOOKUP_CODE.equals(fundCode) ? completedFuture(null) : lookupFundId(DEFAULT_LOOKUP_CODE);
```
Also it would be good to have unit test for such case...

Reply:
That case is already handled in https://github.com/folio-org/mod-gobi/blob/2ee67ab4885e77183b5db4d55498d8c2e03447d6/src/main/java/org/folio/gobi/DataSourceResolver.java#L50. It will look up with the default value from the mapping file in case the first look up is null

Reply:
Thanks for pointing to this!